pkg/standardgadgets/trace/exec/tracer.go

```diff
@@ -15,53 +15,29 @@
 package standard
 
 import (
-	"encoding/json"
-	"fmt"
-
 	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/trace/exec/tracer"
 	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/trace/exec/types"
-	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
 
 	"github.com/kinvolk/inspektor-gadget/pkg/standardgadgets/trace"
 )
 
 type Tracer struct {
-	trace.StandardTracerBase
-
-	eventCallback func(types.Event)
+	trace.StandardTracerBase[types.Event]
 }
```

Comment:
Do we need to keep these structs? I think it's not needed and we can just return `*trace.StandardTracerBase[types.Event]` on `NewTracer()`

Reply:
Applied!